menu.py

```diff
@@ -0,0 +1,31 @@
+import pygame 
+
+import sys
```

Comment:
Import ports should be in the order:
built in modules
third party modules
your own modules

So the `import sys` should be above the `import pygame`